cobigen-templates/templates-devon4j/src/main/templates/crud_openapi_angular_client_app/templates/src/app/model/${variables.etoName#lower_case}.ts.ftl

```diff
@@ -7,7 +7,11 @@
  *
  * NOTE: This class is auto generated by CobiGen.
  */
-<#include "model.ts.ftl">
+<#compress>
```

Comment:
Please revert the changes from this file. Most probably you are using an old version.